src/main/java/module/transfer/history/PlayerDetailPanel.java

```diff
@@ -253,7 +254,8 @@ private void refreshPlayerTable(List<PlayerTransfer> values) {
      */
     private void updatePanel() {
         if (playerId > 0) {
-            if (DBManager.instance().getIsSpielerFired(playerId)) {
+            var playerNotes = DBManager.instance().loadPlayerNotes(playerId);
```

Comment:
`loadPlayerNotes` should probably a `Player` method.

Reply:
Player.getNotes is already available. 

Refactoring of module transfer is another project. i don't want to start it now. do not understand yet if players can appear here whose id is known, but who are not stored in the table SPIELER!?!?

Reply:
FAir enough, not a big deal, something to keep in mind.